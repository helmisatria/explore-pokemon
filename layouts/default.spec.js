import { createLocalVue, shallowMount } from '@vue/test-utils'
import { PiniaVuePlugin } from 'pinia'
import defaultLayout from './default.vue'

describe('Layout: defaultLayout', () => {
  let wrapper

  const localVue = createLocalVue()
  localVue.use(PiniaVuePlugin)

  const generateWrapper = () => {
    return shallowMount(defaultLayout)
  }

  beforeEach(() => {
    wrapper = generateWrapper()
  })

  it('should mount component properly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
