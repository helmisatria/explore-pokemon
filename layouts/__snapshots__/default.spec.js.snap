// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout: defaultLayout should mount component properly 1`] = `
<div
  class="max-w-lg min-h-screen shadow-md mx-auto"
>
  <onlineofflineindicator-stub />
   
  <nuxt />
</div>
`;
